### PR TITLE
MM01 stadgar

### PR DESCRIPTION
**Pull request by andreedd**

### CODE REVIEW

**Comment by andreedd** [comment]
These were approved on höstmöte 2023 and on MM01  15.02.2024

**Comment by andreedd** [comment]
Replaced #9

File: stadgar.md [hunk 1]
```diff
@@ -12,6 +12,8 @@ Antagna på föreningens grundlande möte den 24 augusti 1999
 
 (ändrade och godkändes på föreningens vårmöte 26.03.2021)
 
+(justerade 15.02.2024)
+
```
Comment (JoarSabel) [inline stadgar.md]: We should add the 2022 revision history here

**Review by JoarSabel — COMMENTED** [review]
Looks good, there's not that much else that can be changed xD

**Pull request merged**